plotly/plotly/plotly.py

```diff
@@ -1400,9 +1400,6 @@ def add_share_key_to_url(plot_url, attempt=0):
 
 
 def _send_to_plotly(figure, **plot_options):
-    """
-
-    """
```

Comment:
This has been an empty comment for apparently one year. Should I attempt writing something in it and get it corrected?